server/controllers/orderItemController.js

```diff
@@ -0,0 +1,49 @@
+const OrderItemModel = require('../Models/orderItem');
+const PizzaModel = require('../Models/pizza');
+
+module.exports = {
+    createOrder : async (req, res) => { 
+        PizzaModel.findOne({ _id : req.body.pizza_id })
+            .exec()
+            .then(pizza => {

```

Comment:
Any thoughts on async await methods?

Reply:
They are an option I could use here instead of the chaining I am currently utilizing.

Reply:
Since that is not your default, is it because you are more used to other languages? @mutebironald